internal/server/data/migrations.go

```diff
@@ -734,7 +734,7 @@ func removeDeletedIdentityProviderUsers() *migrator.Migration {
 
 func addUpdateIndexAndGrantNotify() *migrator.Migration {
 	return &migrator.Migration{
-		ID: "2022-09-21T13:50",
+		ID: "2022-10-17T18:00",
```

Comment:
This migration is not in prod yet, but it did run in dev. I'm changing it here instead of creating a new one because these new changes should effectively replace all of the old migration.